examples/wallet-dapi-protectcave.js

```diff
@@ -1,18 +1,26 @@
+const DAPIClient = require('@dashevo/dapi-client');
+
 const { Wallet, EVENTS } = require('../index');
 
+const transport = new DAPIClient({
+  seeds: [{ service: '18.237.69.61:3000' }],
+  timeout: 20000,
+  retries: 5,
+});
 const wallet = new Wallet({
   mnemonic: 'protect cave garden achieve hand vacant clarify atom finish outer waste sword',
   network: 'testnet',
+  transport,
 });
 
 const account = wallet.getAccount();
 const start = async () => {
-  console.log('Balance Conf', await account.getBalance(false, false));
-  console.log('Balance Unconf', await account.getBalance(true, false));
+  console.log('Balance Conf', await account.getConfirmedBalance(false, false));
+  console.log('Balance Unconf', await account.getUnconfirmedBalance(true, false));
   console.log('New Addr', await account.getUnusedAddress().address);
 };
 account.events.on(EVENTS.GENERATED_ADDRESS, (info) => { console.log('GENERATED_ADDRESS'); });
-account.events.on(EVENTS.BALANCE_CHANGED, (info) => { console.log('Balance Changed', info, info.delta); });
+account.events.on(EVENTS.CONFIRMED_BALANCE_CHANGED, (info) => { console.log('Balance Changed', info, info.delta); });
```

Comment:
'Confirmed balance changed' or 'CONFIRMED_BALANCE_CHANGED'

Reply:
Second one for consistency :)